maestro-ios-xctest-runner/maestro-driver-iosUITests/Routes/Handlers/ViewHierarchyHandler.swift

```diff
@@ -0,0 +1,182 @@
+import FlyingFox
+import XCTest
+import os
+
+extension NSException: Error {}
+
+@MainActor
+struct ViewHierarchyHandler: HTTPHandler {
+
+    let maxDepth = 60
```

Comment:
some clarification on why value 60 was picked would be nice in here

Reply:
Turns out we don't need to set the maxDepth anyway